src/docs/product/session-replay/getting-started.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Set Up
+sidebar_order: 1
+redirect_from:
+  - /session-replay/setup/
+  - /session-replay/getting-started/
+description: "Get started with Session Replay, TODO."
+---
+
+<Note>
+
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+
+</Note>
+
+To set up the session replay integration, use the links for supported SDKs below. TODO.
+
+## Supported SDKs
+
+- JavaScript
+  - [Angular](/platforms/angular/session-replay/)
```

Comment:
Not sure these links will work. They have to render as `platforms/javascript/guide/xyz/session/replay`, usually requires some components `<Platform ..` in some cases

Reply:
removed